fix: csv parser delimiter (#1774)
actualbudget/actual: upcoming-release-notes/1774.md

---
category: Bugfix

> authors: [[KaiBelmo](https://github.com/KaiBelmo)]

Reviewer: <!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
authors: [KaiBelmo]
```

when we publish a release o nthe main website the github author links get automatically added. No need to do this manually.

---

> Now you can select delimiters in CSV options when you're uploading a CSV, and it's going to apply to parsing. I've also added a new delimiter '|'.

Reviewer: <!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
Fix selecting delimiters in CSV options when uploading a CSV; it will apply to parsing. Also added a new delimiter '|'.
```
